### DISCUSSION TITLE
Thank you for this build... question from Mac user...

### DISCUSSION THREAD

**Discussion by chadsh3** [flowdriveai/flowpilot#18] [General]
Hi, glad to see someone ported this to Android.  looking forward to try it.  Never used Ubuntu.  Anyone has a command (step-by-step) for Mac with M1?  I'll be glad to buy coffee

**Comment by MankaranSingh** [comment]
This is going to be a long process. The libraries in `libs` folder would be needed to be built for mac. If we can get those then further steps should be easier.

**Reply by iamshreeram** [reply]
@chadsh3 , @MankaranSingh - If the underlying software used in flowpilot is identical to comma, this might be useful: https://x.com/ryansdeth/status/1742665557537890624?s=46.
